modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -135,6 +164,52 @@ def test_blobFromImage(self):
         target = target.transpose(2, 0, 1).reshape(1, 3, height, width)  # to NCHW
         normAssert(self, blob, target)
 
+    def test_model(self):
+        img_path = self.find_dnn_file("dnn/dog416.png")
+        weights = self.find_dnn_file("dnn/resnet50_rfcn_final.caffemodel")
+        config = self.find_dnn_file("dnn/rfcn_pascal_voc_resnet50.prototxt")
+        frame = cv.imread(img_path)
+        model = cv.dnn.DetectionModel_create(weights, config)
+        size = (800, 600)
+        mean = (102.9801, 115.9465, 122.7717)
+        model.setParams(width=size[0], height=size[1], mean=mean)
+
+        iouDiff = 0.011
+        confThreshold = 0.8
+        nmsThreshold = 0
+        scoreDiff = 4e-3
+
+        classIds, confidences, boxes = model.detect(frame, confThreshold, nmsThreshold)
+
+        refClassIds = (7, 12)
+        refConfidences = (0.991359, 0.94786)
+        refBoxes = ((491.822, 81.1668, 211.751, 98.0672),
+                    (132.093, 223.903, 206.984, 343.257))
+
+        normAssertDetections(self, refClassIds, refConfidences, refBoxes,
+                             classIds, confidences, boxes,
+                             confThreshold, scoreDiff, iouDiff)
+
+        frame = cv.resize(frame, size)
+        for box, conf, classId in zip(boxes, confidences, classIds):
+            left, top, width, height = box
+            left = int(left)
```

Comment:
I cannot think up an application where we really need bounding box coordinates in double precision. So for now we have a method which produces `cv::Rect2d` that means that [to draw](https://docs.opencv.org/master/d6/d6e/group__imgproc__draw.html#gac865734d137287c0afb7682ff7b3db23) these rectangle on the image or even [crop a ROI](https://docs.opencv.org/master/d3/d63/classcv_1_1Mat.html#a4272fce43d7405603fbb5e5f52580e57) we need to cast it to `cv::Rect` (which is `cv::Rect2i`).

So  think that we need to remove `absoluteCoords` flag at `Model::detect` and return bounding boxes in the absolute coordinates with integer data type. What do you think?